src/components/metadata/MetadataToolbarItems.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { mockSchema } from './TestUtil';
+import * as api from '@kaoto/api';
+import { MetadataToolbarItems } from '@kaoto/components';
+import { screen } from '@testing-library/dom';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+
+jest.mock('@kaoto/api', () => {
+  return {
+    __esModule: true,
+    ...jest.requireActual('@kaoto/api'),
```

Comment:
mmm I wonder what happened here, since we're not mocking any API :thinking:

Reply:
This is a "charm" I found at somewhere, without this the later spyOn didn't work - https://github.com/KaotoIO/kaoto-ui/pull/1951/files/608732269e75d09be304cd20e1d2468378a6ae04#diff-f37f26c5fb3f0d3c46bdfb42592db926f99f7d4db3bdc487958b5a7ac1c8c273R16
I barely remenber the post says `__esModule: true` is the key, not sure what it means :sweat_smile: